riff-raff.yaml

```
@@ -0,0 +1,11 @@
+regions:
+  - eu-west-1
+stacks:
+  - about-us
```

```suggestion
  - membership-about-us
```

This should make RiffRaff happy to deploy this.

RiffRaff has config for credentials to use for which deployment. The config looks like this:

```json
{"stack" : "frontend", "app" : ".*", "stage" : ".*", "value" : "arn:aws:iam::AAAA:role/BBBB", "comment" : "frontend cross account role"},
{"stack" : "memb.*", "app" : ".*", "stage" : ".*", "value" : "arn:aws:iam::XXXX:role/service/YYYY", "comment" : "membership cross account role"},
```

As there is no line for `"stack": "about-us"`, RiffRaff correctly reported that it could not find credentials when we tried to deploy the app.

Two ways we can fix this:
1. Update RiffRaff's config with this new stack
2. Prefix/namespace this new deployment

As you can see, the config uses regex. That is, for any stack starting "memb" it will use the role `arn:aws:iam::XXXX:role/service/YYYY`. Therefore, option 2 is easiest and provides a little more clarity in terms of ownership of the app.

I've suggested this change above.

Hooray for wildcards in regex 🎉 !